cli/src/lib/__mocks__/node.js

```diff
@@ -13,7 +13,7 @@ export const fs = {
   // This is a custom function that our tests can use during setup to specify
   // what the files on the "mock" filesystem should look like when any of the
   // `fs` APIs are used.
-  mockFiles: {},
+  mockFiles: ({ ...null }: { [key: string]: any }),
```

Comment:
Looks like this breaks the CI, also could you add a comment, that `{...null}` is not straightforward...

Reply:
actually don't need `...null` anymore. Needed it before when it was untyped and was exported an as unsealed object